packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -185,8 +185,12 @@ function build(syscall, _state, makeRoot, forVatID) {
 
   function importPromise(vpid) {
     insistVatType('promise', vpid);
+    assert.equal(
+      parseVatSlot(vpid).allocatedByVat,
```

Comment:
Nit: Instead of `assert.equal(thing, false, whinge)` it seems better to write `assert(!thing, whinge)`

Reply:
aye, will fix